src/uct/ib/rdmacm/rdmacm_component.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017-2021.  ALL RIGHTS RESERVED.
+ * Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "rdmacm_cm.h"
+
+static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
+    {"", "", NULL,
```

Comment:
CM_

Reply:
handled in https://github.com/openucx/ucx/pull/6123, along with tcp_sockcm

Reply:
@evgeny-leksikov merge issue? should be CM_